api-specificatie/common.yaml

```diff
@@ -18,6 +18,12 @@ components:
       description: CRS van de meegegeven geometrie.
       schema:
         type: string
+        enum:
+          - epsg:28992
+        default:
+          - epsg:28992
+# Dit is RD, het enige coordinatenstelsetl dat het kadaster nu kan leveren en daarmee dus de default (ook impliciet indien niet gedefinieerd als default).
```

Comment:
Kan je de mapping van de enumeratiewaarde op omschrijving RD niet in de description zetten?

Reply:
Dat kan zeker. Ik heb het commentaar toegevoegd om de conclusie van de discussie die we gehad hebben direct bij de hand te hebben. Zal ik van de omschrijving het volgende maken : 
"CRS van de meegegeven geometrie. epsg:28992 mapt op het RD New Nederlands coordinatenstelsel.